app/metrics.yaml

```diff
@@ -2966,73 +2966,6 @@ private_browsing_mode:
       - fenix-core@mozilla.com
     expires: "2021-08-01"
 
-contextual_hint.tracking_protection:
-  display:
-    type: event
-    description: |
-      The enhanced tracking protection contextual hint was
-      displayed.
-    bugs:
-      - https://github.com/mozilla-mobile/fenix/issues/9625
-    data_reviews:
-      - https://github.com/mozilla-mobile/fenix/pull/11923
-      - https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877
-      - https://github.com/mozilla-mobile/fenix/pull/18143
-    data_sensitivity:
-      - interaction
-    notification_emails:
-      - fenix-core@mozilla.com
-    expires: "2021-07-01"
-  dismiss:
-    type: event
-    description: |
-      The enhanced tracking protection contextual hint was
-      dismissed
-      by pressing the close button
-    bugs:
-      - https://github.com/mozilla-mobile/fenix/issues/9625
-    data_reviews:
-      - https://github.com/mozilla-mobile/fenix/pull/11923
-      - https://github.com/mozilla-mobile/fenix/pull/13958#issuecomment-676857877
-      - https://github.com/mozilla-mobile/fenix/pull/18143
-    data_sensitivity:
-      - interaction
-    notification_emails:
-      - fenix-core@mozilla.com
-    expires: "2021-07-01"
-  outside_tap:
-    type: event
```

Comment:
As we are removing these metrics, for the old UI, should we talk to product about adding new telemetry for the new UI.

Reply:
I'm not sure if we are covering the new UI with other metrics